docs/sources/operations/loki-ui/_index.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Loki Deployment UI 
+menuTitle: Loki UI
+description: Describes how to setup and use the Loki Deployment UI
+weight: 100
+---
+# Loki Deployment UI 
+
+{{< admonition type="warning" >}}
+The Loki Deployment UI is experimental and subject to change. Before attempting to deploy the Loki make sure to consult the [Considerations](#considerations) section.
+{{< /admonition >}}
+
+The Loki Deployment UI is a web-based user interface that allows you to manage and monitor a Loki deployment in [microservice mode](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#microservices-mode) or [single binary mode](https://grafana.com/docs/loki/<LOKI_VERSION>/get-started/deployment-modes/#monolithic-mode).
+
+{{< figure max-width="100%" src="/media/docs/loki/loki-ui.png" caption="Loki UI Home Page" >}}
+
+## Accessing the Loki Deployment UI
+
+The Loki Deployment UI is available at the following URL:
+
+```
+http://<LOKI_HOST>:<LOKI_PORT>/ui
+```
+
+Where `<LOKI_HOST>` is the hostname or IP address of the Loki server and `<LOKI_PORT>` is the port number that Loki is running on. By default, Loki runs on port `3100`. If you are running Loki in Kubernetes the UI is made accessible via the `loki-gateway` service. Either by port-forwarding or by exposing the service via a LoadBalancer. 
+
+{{< admonition type="caution" >}}
+`/ui` currently cannot be changed to another path. This would require further contribution.
+{{< /admonition >}}
+
+## Deployment
+
+Each Loki component now includes the UI service. This needs to be enabled in the Loki configuration. The UI service is disabled by default. To enable the UI service, add the following configuration to the Loki configuration file:
```

Comment:
Enabling just configure correct service discovery btw, you can't disable the UI right now.

Not sure we need the precision, but basically by default they act as single nodes.